tagmanager/ctags/parse.h

```diff
@@ -128,7 +128,8 @@ extern void checkRegex (void);
 
 /* Extra stuff for Tag Manager */
 extern tagEntryFunction TagEntryFunction;
-extern void setTagEntryFunction(tagEntryFunction entry_function);
+extern void *TagEntryUserData;
```

Comment:
Yeah, agree, just for now I tried to make minimal changes to ctags as I want to sync them with uctags anyway and do the "right" stuff once the sync is done.